apps/vessel-history/features/vessels/activity/vessels-activity.selectors.ts

```diff
@@ -112,11 +112,44 @@ export const selectEventsForTracks = createSelector(
   }
 )
 
+// This selet will split the gaps in two events before add the rendering data
+export const selectEventsWithGapsSplit = createSelector([selectEventsForTracks], (events) => {
+  return events // I dont know if this change is temporaly
```

Comment:
@fertandil87 if this selector only returns events it's not needed, please remove it.